src/platform-apps/channels/channel-list.test.tsx

```diff
@@ -53,4 +54,33 @@ describe('ChannelList', () => {
       'four',
     ]);
   });
+
+  it('renders category name', () => {
+    const channels = [
+      { id: 'one', name: 'first channel', category: 'catg1' },
+      { id: 'two', name: 'second channel', category: 'catg1' },
+      { id: 'three', name: 'third channel', category: 'catg2' },
+      { id: 'four', name: 'fourth channel', category: 'catg2' },
+    ];
+
+    const wrapper = subject({ channels });
+    expect(wrapper.find('.channel-list__category-channel-name').exists()).toBe(true);
+  });
+
+  it('renders grouped category', () => {
+    const channels = [
+      { id: 'one', name: 'first channel', category: 'catg1' },
+      { id: 'two', name: 'second channel', category: 'catg1' },
+      { id: 'three', name: 'third channel', category: 'catg2' },
+      { id: 'four', name: 'fourth channel', category: 'catg2' },
```

Comment:
It looks like all these tests depend on the same test data. Rather than redefining channels for each test can we move channels into the `subject`? 

```
  const subject = (props: any = {}) => {
     const channels = [
        { id: 'one', name: 'first channel', category: 'catg1' },
        { id: 'two', name: 'second channel', category: 'catg1' },
        { id: 'three', name: 'third channel', category: 'catg2' },
        { id: 'four', name: 'fourth channel', category: 'catg2' },
     ];
    const allProps = {
      channels,
      ...props,
    };
    return shallow(<ChannelList {...allProps} />);
  };
```